### PR TITLE
Add the information on the WoT Japanese CG to wot-marketing/cg

### PR DESCRIPTION
**Pull request by ashimura**
resolves issue #121

### CODE REVIEW

**Review by k-toumura — APPROVED** [review]
Thank you, @ashimura .   It looks good to me.
We should upload a CG charter text as soon as possible...

**Comment by ashimura** [comment]
@k-toumura that is a good question :)
Meanwhile, I've just changed the index.html to refer to the [WoT-JP CG page](https://www.w3.org/community/wot-jp/) for the "CG Scope" link tentatively.

We can change the URL again once the CG Charter is installed appropriately :)

File: docs/cg/index.html [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+title: W3C WoT – Community Groups
 group: "navigation"
 ---
 <div>
-	<h2>W3C WoT – Community Group</h2>
+	<h2>W3C WoT – Community Groups</h2>
+	<p>There are the following two Community Groups:</p>
+	<ul>
+		<li>Web of Things Commuity Group</li>
+		<li>Web of Things Japanese Commuity Group</li>
```
Comment (danielpeintner) [inline docs/cg/index.html:11]: twice a minor type --> "Community"

**Review by danielpeintner — CHANGES_REQUESTED** [review]
Looks good. Just 2 minor typos that we should fix before merging.

File: docs/cg/index.html (hunk 1 above)
Comment (ashimura) [reply docs/cg/index.html]: typos have been fixed :)

**Comment by sebastiankb** [comment]
I had a brief chat with @ashimura and it's ok to merge this PR.

**Pull request merged**